src/petals/server/throughput.py

```diff
@@ -112,14 +112,18 @@ def measure_compute_rps(
     dtype: torch.dtype,
     *,
     load_in_8bit: bool,
+    tensor_parallel_devices: Sequence[torch.device],
```

Comment:
Please add `tensor_parallel_devices` param to `get_host_throughput()`, pass it there, and also add it to `cache_key` (if present)

Reply:
done, thanks!

Reply:
You haven't added it to `cache_key`. Now, throughput won't be measured again if one enables tensor parallelism, the server will use the old throughput. Please fix this.